ClassLibrary1/Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+namespace ClassLibrary1
```

Comment:
Delete

Reply:
Its gone, I'll explain why it even appeared